packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -63,34 +65,37 @@ export class CrossmintAASDK {
 
             const userIdentifier = parseUserIdentifier(user);
 
-            const publicClient = createPublicClient({
-                chain: getViemNetwork(chain as EVMBlockchainIncludingTestnet),
-                transport: http(getBundlerRPC(chain)),
-            });
-              
             const owner = await createOwnerSigner({
                 chain,
                 walletConfig,
             });
 
             const address = owner.address;
 
+            const publicClient = createPublicClient({
+                //chain: getViemNetwork(chain as EVMBlockchainIncludingTestnet),
+                transport: http(getBundlerRPC(chain)),
+            });
+
             const ecdsaValidator = await signerToEcdsaValidator(publicClient, {
                 signer: owner,
+                entryPoint: ENTRYPOINT_ADDRESS_V07,
             });
 
             const account = await createKernelAccount(publicClient, {
                 plugins: {
                     sudo: ecdsaValidator,
                 },
+                index: BigInt(0),
```

Comment:
can you put the link to the doc where it mentions adding this index?

Reply:
https://docs.zerodev.app/sdk/presets/zerodev#ecdsa-validator

<img width="787" alt="Captura de pantalla 2024-04-17 a las 9 37 48" src="https://github.com/Crossmint/crossmint-sdk/assets/131860331/7c24fcd7-3194-4a7a-8ffb-8f28f31f44f9">